signer/contentsignaturepki/x509.go

```diff
@@ -37,6 +37,10 @@ func (s *ContentSigner) findAndSetEE(conf signer.Configuration) (err error) {
 		err = errors.Wrapf(err, "found suitable end-entity labeled %q in database but not in hsm", s.eeLabel)
 		return
 	}
+	if len(s.PublicKey) < 50 {
+		err = errors.Errorf("malformed public key in key label %q for signer %q: %q", s.eeLabel, s.ID, s.PublicKey)
```

Comment:
Worth mentioning how the keys is malformed in the error message.

Reply:
changed the message to mention it, but at that point we don't know _why_ it's malformed

Reply:
well why we're considering it malformed